mod_reforged/hooks/config/strings.nut

```diff
@@ -1021,6 +1022,16 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+ 	RF_Dismemberment = ::UPD.getDescription({
+ 		Fluff = "Welcome to the chopping block!",
+ 		Requirement = "Axe",
+ 		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"When inflicting [injuries|Concept.InjuryTemporary] with an attack, always inflict one with the highest [damage threshold|Concept.InjuryThreshold] even if the damage did not meet that [threshold|Concept.InjuryThreshold]."
```

Comment:
You force-deleted the second effect of this perk; the one about triggering morale checks on the enemy.